src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+
+/**
+ * Project Duke is a educational software project designed to take you
+ * through the steps of building a small software incrementally,
+ * while applying as many Java and SE techniques as possible along the way.
+ *
+ * @author AdrianOngJJ
+ * @version 0.1
+ * @since 22/1/2022
+ */
 public class Duke {
-    public static void main(String[] args) {
+    private static final String LINE_BREAK = "\n~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    private static final ArrayList<Task> masterList = new ArrayList<>();
+    /**
+     * Prints line break.
+     * @return void
+     */
+    private static final void printLineBreak() {
+        System.out.println(LINE_BREAK);
+    }
+
+    /**
+     * Prints Master List
+     * @param bw BufferedWriter from main to print out the Master List.
+     * @throws java.io.IOException If an I/O error occurs. Only takes in string.
+     */
+    private static final void printList(BufferedWriter bw) throws Exception {
+        for(int i = 0; i < masterList.size(); i++) {
+            Task curr = masterList.get(i);
+            bw.write((i + 1) + "." + curr.toString());
+            bw.newLine();
+        }
+        bw.flush();
+    }
+
+    private static final String getDateTime(String[] inputArr) {
+        return inputArr[1].split("/")[1].split(" ", 2)[1]; // split input from slash
```

Comment:
Perhaps include the comment as a same-level indent above the line as opposed to an inline one? I noticed this in other places as well. 

Since this function consists of a single line at the moment, you might want to add Javadoc-style documentation instead.